### ISSUE TITLE
cross-build: PR 51357 causing cross build failure

### ISSUE THREAD

**Issue by cblecker** [kubernetes/kubernetes#51815]
```
I0901 16:54:03.182] # k8s.io/kubernetes/pkg/kubelet/cm
I0901 16:54:03.182] pkg/kubelet/cm/container_manager_windows.go:34: cannot use containerManagerImpl literal (type *containerManagerImpl) as type ContainerManager in assignment:
I0901 16:54:03.182] 	*containerManagerImpl does not implement ContainerManager (wrong type for Start method)
I0901 16:54:03.183] 		have Start(*"k8s.io/kubernetes/vendor/k8s.io/api/core/v1".Node, ActivePodsFunc) error
I0901 16:54:03.183] 		want Start(*"k8s.io/kubernetes/vendor/k8s.io/api/core/v1".Node, ActivePodsFunc, status.PodStatusProvider, cri.RuntimeService) error
I0901 16:54:03.183] pkg/kubelet/cm/container_manager_windows.go:42: cannot use containerManagerImpl literal (type *containerManagerImpl) as type ContainerManager in return argument:
I0901 16:54:03.183] 	*containerManagerImpl does not implement ContainerManager (wrong type for Start method)
I0901 16:54:03.184] 		have Start(*"k8s.io/kubernetes/vendor/k8s.io/api/core/v1".Node, ActivePodsFunc) error
I0901 16:54:03.184] 		want Start(*"k8s.io/kubernetes/vendor/k8s.io/api/core/v1".Node, ActivePodsFunc, status.PodStatusProvider, cri.RuntimeService) error
```
https://github.com/kubernetes/kubernetes/pull/51357
https://k8s-testgrid.appspot.com/misc#cross-build
https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/logs/ci-kubernetes-cross-build/3131

---
cc: @kubernetes/sig-node-bugs @kubernetes/sig-node-test-failures @ConnorDoyle @derekwaynecarr @smarterclayton 
/kind bug
/priority failing-test

**Comment by dims** [comment]
Looks like this one broke things https://github.com/kubernetes/kubernetes/commit/7c6e31617d70c4ed161d63bc1358754dcb7c2910

**Comment by ConnorDoyle** [comment]
This is already in the process of being reverted. See #51804

**Comment by cblecker** [comment]
Fixed by revert in #51804
